Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
This would still require the user calling `with_temporary_scope` before being able to use mocks correct?